app/src/main/java/de/westnordost/streetcomplete/data/meta/OsmTaggings.java

```diff
@@ -22,4 +22,9 @@
 	};
 
 	public static final String SURVEY_MARK_KEY = "check_date";
+
+	public static final String[] OFFICES_VISITED_BY_GENERAL_PUBLIC = {
+		"insurance", "government", "lawyer", "estate_agent", "political_party", "travel_agent",
+		"tax_advisor", "therapist", "religion",
+	};
```

Comment:
# Opening hours

lawyers are not really walk-in for customers, as much as an architect or other company providing a service (such as: building a table, building a smartphone app) aren't. They do work for clients, but the office hours are certainly not opening hours.

Example: [White & Case](https://www.whitecase.com/locations/emea/hamburg) (big law firm) lists no opening hours, only a phone number.

Same with political parties, it is not really public in any sense. You can't simply walk in there, it is rather the point where the political party meets. Maybe they organize some public Q&A hours, maybe the don't.

Also, from my experience, therapists do not have a reception, thus, it's not walk-in and appointments are made via phone or internet.

Regarding tax advisors, I am not sure. The offices I know *look like* one could just walk in there from the outside, but I am not sure if they really have opening hours.

# Name

✔️ 

# Wheelchair

Well, nothing changed here. Some are not walk-in, but so is f.e. a concert hall or theatre, and `wheelchair` is and should be tagged on these as well.